secure_headers.gemspec

```diff
@@ -9,12 +9,12 @@ Gem::Specification.new do |gem|
   gem.version       = SecureHeaders::VERSION
   gem.authors       = ["Neil Matatall"]
   gem.email         = ["neil.matatall@gmail.com"]
-  gem.description   = "Manages application of security headers with many safe defaults."
-  gem.summary       = 'Add easily configured security headers to responses
+  gem.summary       = "Manages application of security headers with many safe defaults."
+  gem.description   = 'Add easily configured security headers to responses
     including content-security-policy, x-frame-options,
     strict-transport-security, etc.'
   gem.homepage      = "https://github.com/twitter/secureheaders"
-  gem.license       = "Apache Public License 2.0"
+  gem.license       = "Apache-2.0"
```

Comment:
I just noticed that https://github.com/github/secure_headers/blob/main/LICENSE shows it's MIT. So what is it?

Reply:
Looks like in https://github.com/github/secure_headers/pull/315 it was relicensed to MIT but they forgot things. https://github.com/github/secure_headers/issues/415 fixed the README (via https://github.com/github/secure_headers/issues/415) but still forgot the gem metadata.